runtime/vm/jvmfree.c

```diff
@@ -27,6 +27,9 @@
 #include "omrlinkedlist.h"
 #include "j9protos.h"
 #include "j9port.h"
+#if defined(J9VM_OPT_SNAPSHOTS)
+#include "j9port_generated.h"
+#endif /* defined(J9VM_OPT_SNAPSHOTS) */
```

Comment:
Why?

Reply:
Odd. I distinctly remember removing this in a previous revision. It must've slipped back in during a squash at some point. Thanks for catching that.